src/integration-test/java/com/tlswe/awsmock/cloudwatch/CloudWatchBaseTest.java

```diff
@@ -159,8 +156,12 @@ protected final Datapoint getMetricStaticticsTest(String metricName) {
 
         if (result != null && !result.getDatapoints().isEmpty()) {
             dataPoint = result.getDatapoints().get(0);
+
+            for (Datapoint dp : result.getDatapoints()) {
+                System.out.println(dp.getTimestamp());
```

Comment:
Such debug println may need to be commented or it can pollute the build/test log.